### DISCUSSION TITLE
Is it possible to turn off automatic tooltips for selection?

### DISCUSSION THREAD

**Discussion by vaniwiki** [combostrap/combo#92] [General]
I like automatic tooltips, showing relevant information across my website. But they should not be everywhere. For example, I have a long list of links that do not need any link tooltips, but just remain as links. Is there any way to turn this feature off for certain parts, like a box or card with some syntax like `<box tooltip="none">` or similar...?

**Comment by gerardnico** [comment]
There is a preview attribute. May be?

`[[?preview=false|without preview of the actual page]]`

**Comment by vaniwiki** [comment]
That would be great if it works. But my links with anchor hashtags don't work...

`[[a#Abduction?preview=false|Abduction]]` produces the link `/glossary/a_abduction`

**Reply by gerardnico** [reply]
Anchor must be at the end

**Reply by vaniwiki** [reply]
If I do this `[[a?preview=false#Abduction|Abduction]]`, then previews are back...

**Reply by gerardnico** [reply]
Yeah. Thanks.

It’s a Boolean attribute. It means that in the code, we check only its presence and not its value.

A new feature, improvement then.

**Reply by vaniwiki** [reply]
I have several thousand links on a page, and just want it to be fast and effective. And since the link previews anyways don't show anchor sections, but repeats the main page, then they are of no use in this scenario.

**Reply by gerardnico** [reply]
Yeah. I created an enhancement issue.
